mcda/mcda_functions/aggregation.py

```diff
@@ -66,16 +61,16 @@ def geometric(self, norm_indicators: pd.DataFrame()) -> np.ndarray:
 
         return scores
 
-    def harmonic(self, norm_indicators: pd.DataFrame()) -> np.ndarray:
+    def harmonic(self, norm_indicators: pd.DataFrame) -> np.ndarray:
         """
-        The weighted harmonic mean works only with strictly positive
-        normalized indicator values (i.e. not with minmax, and target with feature range (0,1);
-        not with standardized with feature range ('-inf','+inf')).
-        Gets as input the normalized values of the indicators in a matrix
-        and estimates the scores over the indicators, per alternative.
-
-        :gets: pd.DataFrame() of shape (no.alternatives x num. indicators)
-        :returns: pd.Series of length = num. of alternatives
+        The weighted harmonic mean works only with strictly positive normalized indicator values
+        (i.e. not with minmax, and target with feature range (0,1); and not with standardized with feature range
+        ('-inf','+inf')). It gets as input positive normalized values of the indicators in a matrix and estimates the
+        scores over the indicators, per alternative. The norm_indicators has shape = (no.alternatives x num. indicators)
+        and the returned scores has length = num. of alternatives.
+
+        :param norm_indicators: pd.DataFrame
```

Comment:
Maybe `Parameters` instead of `param` similar to what you added in `utils`. Just to keep similar documentation everywhere.

Also, short description.